src/lib/layers/editable-geojson-layer.js

```diff
@@ -123,6 +126,7 @@ export default class EditableGeoJsonLayer extends EditableLayer {
 
     layers = layers.concat(this.createPointLayers());
     layers = layers.concat(this.createDrawLayers());
+    layers = layers.concat(this.createHintPointLayer());
```

Comment:
No need for a new layer, we already have an edit handle layer with callbacks that allow applications to customize edit handles (e.g. change the color, change to icons, etc).